redfish/eventservice.go

```diff
@@ -7,11 +7,13 @@ package redfish
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/stmcginnis/gofish/common"
+	oemDell "github.com/stmcginnis/gofish/oem/dell"
+	oemHp "github.com/stmcginnis/gofish/oem/hpe"
+	oemZt "github.com/stmcginnis/gofish/oem/zt"
```

Comment:
Please remove circular relationships.

Reply:
SubmitTestEvent() is implemented differently for three oem vendors. They use different http request payloads and respond with different "OK" status codes.

The current (SubmitTestEvent)[https://github.com/stmcginnis/gofish/blob/main/redfish/eventservice.go#L334) function does not work for all vendors. I would ask to use the [SendTestEvent](https://github.com/nwaizer/gofish/blob/ZTsystem/redfish/eventservice.go#L377) in order to send this request with the payload each vendor accepts.

Reply:
Same as above, implementations for the OEM specific calls need to move into the OEM specific code.

Reply:
circular relationships removed in updated commit